noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/event.nr

```diff
@@ -85,6 +85,39 @@ where
     plaintext_bytes
 }
 
+/// Computes an encrypted log for an event using AES-128 encryption in CBC mode.
```

Comment:
Sneaked this unrelated change in. I needed it to orient myself in the structure of the log and AI is perfect for generating these code docs.